src/components/structures/auth/SetupEncryptionBody.tsx

```diff
@@ -89,7 +89,7 @@ export default class SetupEncryptionBody extends React.Component<IProps, IState>
     private onVerifyClick = (): void => {
         const cli = MatrixClientPeg.get();
         const userId = cli.getSafeUserId();
-        const requestPromise = cli.requestVerification(userId);
```

Comment:
So we assume that the crypto is defined here ?

Can we use instead `cli.getCrypto()?.requestOwnUserVerification()` ?